README.md

```diff
@@ -1,2 +1,41 @@
 # rmf_api_msgs
-Collection of messages which bridges the C++ components of RMF to the web interface
+
+Collection of json message schemas which bridges the C++ and python components of RMF to the web interface
+
+# Usage
+
+Download pkg for python model generation
+```bash
+pip3 install datamodel-code-generator
+```
+
+Compile with [colcon](https://colcon.readthedocs.io/en/released/) is recommended
+
+```bash
+colcon build --packages-select rmf_api_msgs
+source install/setup.bash
+```
+
+Subsequently, to access the schemas, the user just needs to include/import the cpp header or py module into their source code.
+
+1. C++ example:
+
+```cpp
+#include <rmf_api_msgs/schemas/task_state.hpp>
+
+nlohmann::json schema = rmf_api_msgs::schemas::task_state
+```
+
+1. Python example:
+
+```py
+from rmf_api_msgs import schemas
+from rmf_api_msgs.models import task_state
```

Comment:
I'm getting the `No module named 'rmf_api_msgs.models'` error when I try this.

Reply:
That's weird. Can u try to `colcon build  --packages-select rmf_api_msgs --event-handlers console_direct+` to check the log output?  Also check if the directory `src/rmf_api_msgs/rmf_api_msgs/rmf_api_msgs/models/` exists?

Reply:
The `models` dir doesn't exist for me.

Reply:
The logs seem fine as well.

Reply:
check if you get the warning msg:  `Not able to generate py models since 'datamodel-codegen' is not avail`. Else can try to install `pip3 install datamodel-code-generator`. Maybe should just fail the compilation if the pkg doesn't exist.

Reply:
Works! mb forgot to check for the dir after installing the codegen and rebuilding